api/issues.py

```diff
@@ -144,54 +174,49 @@ async def issues(
     db: Connection = Depends(database.db),
     langs: LangsNegociation = Depends(langs.langs),
     params=Depends(commons_params.params),
+    i18n: i18n.Translator = Depends(i18n.i18n),
 ) -> Dict[Literal["issues"], List[Dict[str, Any]]]:
-    params.limit = min(params.limit, 100000)
```

Comment:
You lost the custom max limit.

Reply:
Now that `issues()` always relies on `_issues()`, I kept it at https://github.com/osmose-qa/osmose-frontend/pull/536/files#diff-207e11f1ac32911ffb416b8ab839fc02cb1dde93e94b5f4dea2515ffd51262b1R159